common/logisticspipes/gui/GuiLogisticsSettings.java

```diff
@@ -112,6 +115,9 @@ public boolean handleKey(int code, char c) {
 
 		@Override
 		public void guiClose() {
+			Keyboard.enableRepeatEvents(false);
+			getBaseScreen().initGui();
```

Comment:
This (`getBaseScreen().initGui();`) shouldn't have to be done manually, but somewhere in the base screen class every time a subscreen closes.

Reply:
(Reason being, forgetting to call this (or inconsistently calling it based on the closed screen) can cause bugs in other screens and it'll not be immediately obvious what causes it, whereas forgetting to enable repeat events affects only the current GUI)

Reply:
I have already accidentally changed the SubGuiScreen.exitGui() in this commit.
Leave it and only revert SubGuiScreen.initGui() + all TabSubGui.guiClose()?